test-suite/misc/coqc_dash_o.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+$coqc misc/coqc_dash_o.v -o /tmp/coqc_dash_o.vo
```

Comment:
This is insecure and bad practice, at the very least we should use `mktemp` or a local path. I see we have more stuff like this in the test suite.

Reply:
Interesting! I didn't know about `mktemp`.

Reply:
I tried to request changes to my own PR and I discovered I couldn't do it XD